test/unit/TokenController/setup.js

```diff
@@ -32,22 +32,26 @@ async function setup() {
   await master.enrollInternal(internal.address);
   await master.setTokenAddress(nxm.address);
 
-  const masterInitTxs = await Promise.all([
-    master.setTokenAddress(nxm.address),
-    master.setLatestAddress(hex('GV'), governance.address),
-  ]);
-  await Promise.all(masterInitTxs.map(x => x.wait()));
-
   await tokenController.changeMasterAddress(master.address);
   await tokenController.changeDependentContractAddress();
-  await tokenController.connect(internal).changeOperator(tokenController.address);
   await nxm.mint(tokenController.address, ethers.utils.parseUnits('1000'));
 
   for (const member of [...members, tokenController]) {
     await master.enrollMember(member.address, 2);
     await tokenController.connect(internal).addToWhitelist(member.address);
   }
 
+  master.setLatestAddress(hex('GV'), accounts.governanceContracts[0].address);
```

Comment:
what's the reason for this change? @danoctavian

Reply:
as discussed in DMs, it's just so one can execute the changeOperator call which is now restricted to onlyGovernance

Injecting an EOA as GV temporarily to call that for the test setup